src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int maxvalue = 100;
```

Comment:
```suggestion
    private static final int MAX_VALUE = 100;
```
and put this first

Reply:
Static variable definition in wrong order. [DeclarationOrder]

Reply:
![image](https://github.com/user-attachments/assets/ae443d4f-0582-4389-bd1e-36fe2171a0d7)